rmgpy/rmg/react.py

```diff
@@ -43,14 +43,35 @@ def react(spcA, speciesList=[]):
     """
     Generate reactions between spcA and the list of 
     species for all the reaction families available.
+
+    Returns an empty list of the parameter species is non-reactive.
```

Comment:
Plus I think it would be good to mention the name `spcA` (also on line 49, if that's what the 'method parameter species' means.